src/java.desktop/windows/classes/sun/awt/windows/ThemeReader.java

```diff
@@ -150,7 +141,7 @@ private static Long getTheme(String widget, int dpi) {
         // mostly copied from the javadoc for ReentrantReadWriteLock
         Long theme = null;
 
-        if(dpiAwareWidgetToTheme.containsKey(dpi))
+        if (dpiAwareWidgetToTheme.get(dpi) != null)
         {
```

Comment:
```suggestion
        if (dpiAwareWidgetToTheme.get(dpi) != null) {
```
For consistent Java style.